### PR TITLE
Added sample nodejs-example-logs-api-extension

### PR DESCRIPTION
**Pull request by stawecki**
*Issue #, if available:*
Issue #13 - Add Logs API extension in Node.js 

*Description of changes:*
Added a new Lambda Extension sample for using Logs API in Node.js.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/http-listener.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                response.writeHead(200, {})
+                response.end("OK")
+            });
+        } else {
+            console.log('GET');
+            response.writeHead(200, {});
+            response.end("OK");
+        }
+    });
+    server.listen(port);
```
Comment (santiagocardenas) [inline nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/http-listener.js]: Can you please listen on hostname `sandbox`?

File: nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/extensions-api.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        method: 'post',
+        body: JSON.stringify({
+            'events': [
+                'INVOKE',
+                'SHUTDOWN'
+            ],
+        }),
+        headers: {
+            'Content-Type': 'application/json',
+            'Lambda-Extension-Name': basename(__dirname),
```
Comment (santiagocardenas) [inline nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/extensions-api.js:19]: Worth adding a quick note on top of this that the name of the extension must match the file name of the extension itself that's in `/opt/extensions/`, that is `nodejs-example-logs-api-extension`.

File: nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "nodejs-example-extension",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "aws-sdk": "*",
```
Comment (santiagocardenas) [inline nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/package.json]: Best to lock to a version.

**Comment by santiagocardenas** [comment]
@stawecki , overall this looks great and thank you for your contribution! If you can please help us with the comments, we can get this merged as a very neat example of a Node.js Logs API extension :)

Thanks!

**Comment by stawecki** [comment]
@santiagocardenas thanks for having a look! I hope the adjustments above will suffice. Happy to help :)

File: nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/package.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "nodejs-example-extension",
```
Comment (santiagocardenas) [inline nodejs-example-logs-api-extension/nodejs-example-logs-api-extension/package.json]:
```suggestion
  "name": "nodejs-example-logs-api-extension",
```

**Comment by neeleshsaxena** [comment]
Hey folks, had a question on this example. I'm trying to mount this node extension on a Java runtime Lambda, and I'm gettin this error pasted below. Is this because the runtimes are different or am I missing some permissions? 
`nodejs-log-api-extension  launching extension`
`/usr/bin/env: node: No such file or directory`

**Pull request merged**